inc/fields/class-shortcode-ui-field-post-select.php

```diff
@@ -137,11 +137,21 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 
 		$query = new WP_Query( $query_args );
 
+		$is_multiple_post_types = count( $query->get( 'post_type' ) ) > 1;
```

Comment:
I think the `post_type`  should be left to when setting up the setting. For a certain dropdown I may only want posts and in another just pages.